### ISSUE TITLE
Geração de Remessa CNAB400 com 2 ou mais instruções para Santander não quebra a linha

### ISSUE THREAD

**Issue by cetimarcelo** [BoletoNet/boletonet#807]

**Comment by cetimarcelo** [comment]
Eu verifiquei o problema e o mesmo acontece em Banco_Santander.cs, na linha 1632 que tem o seguinte código:

`                _detalhe = Utils.SubstituiCaracteresEspeciais(_detalhe);
`
Alguém já passou por esse problema?

**Comment by diegomodolo** [comment]
Eu arrumei isso, mas não consegui fazer o merge.
Se precisar de forma urgente pode usar a versão do meu fork.

**Comment by carloscds** [comment]
@diegomodolo Precisa de ajuda com o PR ?

**Comment by diegomodolo** [comment]
@carloscds , você tentou me ajudar há um tempo mas não conseguimos, se lembra?
Ah, ontem eu precisei fazer uma alteração na criação da linha digitável também, enviei para homologação e estou aguardando retorno do banco.

**Comment by carloscds** [comment]
@diegomodolo Quer tentar fazer ?

**Comment by diegomodolo** [comment]
Podemos tentar sim, o problema é que meu fork tem outras alterações, e eu não consegui "ignorá-las" ao fazer o PR, então acaba indo mais coisa do que o necessário.

**Comment by carloscds** [comment]
@diegomodolo Você precisa mover as alteracoes pendentes para uma branch e atualizar a master do seu fork.

**Comment by diegomodolo** [comment]
Então, eu não consegui criar um branch a partir do repositório original. Sempre que eu tento acaba vindo meu fork.

**Comment by carloscds** [comment]
@diegomodolo você fez a mudança na sua branch master ?

**Comment by diegomodolo** [comment]
Fiz sim. Eu não entendo muito de git, aqui na empresa usamos TFVC.

**Comment by carloscds** [comment]
@diegomodolo Podemos fazer um call e ajustamos.

**Comment by diegomodolo** [comment]
Claro! Me dê umas duas opções de horário que fica bom para você e eu me organizo.

**Comment by carloscds** [comment]
@diegomodolo Hoje as 16 ou amanha as 14

**Comment by diegomodolo** [comment]
Pode ser hoje 16. Meu Skype é diegomodolo.

**Comment by diegomodolo** [comment]
Um teste falhou no build do meu PR. Esse teste da linha digitável foi feito baseado em uma homologação real? Pois no meu cliente o quarto grupo está sendo rejeitado, ele deve ser o mesmo dígito do código de barras, e como o Mod11 estava errado, o DV do código de barras também estava. 
Como procedo neste caso?

**Comment by brunobertechini** [comment]
FYI: Estou trabalhando na implementação de CNAB240 de Pagamentos a fornecedor em um fork do BoletoNetCore.

Vou ter que alterar um pouco o mapeamento OO pois os conceitos de Beneficiario e Pagador sao diferentes pra boleto e pagamentos a fornecedor. Provavelmente será melhor um approach de subclasse para cada tipo de arquivo de remessa.
